Advanced_Projects/PDFTextGenie/gui/main_window.py

```
@@ -0,0 +1,310 @@
+import sys
+import os
+from PyQt5.QtWidgets import (
+    QApplication, QMainWindow, QWidget, QVBoxLayout, QPushButton, QLabel, QFileDialog, QListWidget, QTextEdit, QProgressBar, QHBoxLayout, QCheckBox, QComboBox, QMessageBox, QSlider, QSpinBox
```

Ruff (0.12.2)

`PyQt5.QtWidgets.QSlider` imported but unused

Remove unused import: `PyQt5.QtWidgets.QSlider`

(F401)